dhis-2/dhis-api/src/main/java/org/hisp/dhis/datastatistics/DataStatistics.java

```diff
@@ -131,6 +136,17 @@ public void setReportTableViews( Double reportTableViews )
         this.reportTableViews = reportTableViews;
     }
 
+    @JsonProperty
+    public Double getVisualizationViews()
+    {
+        return visualizationViews;
+    }
+
+    public void setVisualizationViews( Double visualizationViews )
```

Comment:
This class is mapped in DataStatistics.hbm.xml. I believe we need the getter/setters to keep Hibernate working in the standard way. I'm afraid it could cause some side effects.

Reply:
True